CMakeLists.txt

```diff
@@ -600,15 +600,42 @@ endif(UNIX)
 
 # Copy libraries needed by Windows
 if(WIN32)
+    add_custom_command(TARGET antimicrox POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E echo "dll files linked with antimicrox executable: $<TARGET_RUNTIME_DLLS:antimicrox> "
+        COMMAND_EXPAND_LISTS
+        )
+    add_custom_command(TARGET antimicrox POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_RUNTIME_DLLS:antimicrox> $<TARGET_FILE_DIR:antimicrox>
+        COMMAND_EXPAND_LISTS
+        )
+
+    find_path(SDL2_DLL_LOCATION_DIR "SDL2.dll" HINTS "${SDL2_LIBRARY_DIRS}/../bin")
+
+    #these three lines assume SDL2 in standard location TODO do it better
```

Comment:
@avinal   
Do you know any less messy way of finding these DLL-s?

Reply:
I think the DLL-s can be packaged automatically by CMake. I will look into it and let you know.

Reply:
Well, it seems there is no way to automatically add a third-party library. But I think `file(GLOB ...)` can be a better choice since we are doing repetitive tasks and it can be put in a loop to reduce the number of lines.